python/paddle/fluid/incubate/fleet/parameter_server/pslib/optimizer_factory.py

```diff
@@ -162,7 +162,7 @@ def _minimize(self,
         opt_info["fleet_desc"] = ps_param
         opt_info["worker_skipped_ops"] = worker_skipped_ops
         opt_info["use_cvm"] = strategy.get("use_cvm", False)
-
+        opt_info["dump_slot"] = True
         for loss in losses:
```

Comment:
why dump_slot is default true， and user can not set it from strategy ?  what if a user doesn't need dump_slot

Reply:
done